back/config.yaml

```diff
@@ -1,2 +1,3 @@
 server:
   port: 9876
+  key: lmHW=WhEZLH@$K!11kazTnQY9=rDQct)WhUdqu_Ka6&&K2)JdxJ$B&75)#xmVP_D
```

Comment:
clear key ?

Reply:
This is an example key, as well as the `.env.example` everything is made for dev,

Though I could move it to a `config.yaml.example` and delete this one 😄

Reply:
Yup i think it would be clearer :)